ethsigner/signer/multiplatform/src/main/java/tech/pegasys/ethsigner/signer/multiplatform/MultiSignerFactory.java

```diff
@@ -15,10 +15,13 @@
 import tech.pegasys.ethsigner.core.signing.TransactionSigner;
 import tech.pegasys.ethsigner.signer.multiplatform.metadata.AzureSigningMetadataFile;
 import tech.pegasys.ethsigner.signer.multiplatform.metadata.FileBasedSigningMetadataFile;
+import tech.pegasys.ethsigner.signer.multiplatform.metadata.HashicorpSigningMetadataFile;
 
 public interface MultiSignerFactory {
 
-  TransactionSigner createSigner(AzureSigningMetadataFile metadataFile);
+  TransactionSigner createSigner(final AzureSigningMetadataFile metadataFile);
```

Comment:
apparently "final" on interfaces does nothing ... so can leave them off of here (ask CJH for more info ... I think)